extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/media/GoogleMediaExporter.java

```diff
@@ -216,7 +221,7 @@ private ExportResult<MediaContainerResource> exportPhotosContainer(
     List<IdOnlyContainerResource> subResources = new ArrayList<>();
 
     for (PhotoAlbum album : container.getAlbums()) {
-      GoogleAlbum googleAlbum = getOrCreatePhotosInterface(authData).getAlbum(album.getId());
+      GoogleAlbum googleAlbum = getGoogleAlbum(album.getIdempotentId(), album.getId(), album.getName(), authData);
```

Comment:
Did you want to skip albums that return null the same way you did for photos and videos in [#1295](https://github.com/google/data-transfer-project/pull/1295/files) ie:
`if (googleAlbum == null) {
        continue;
      }`
or did you intentionally not do that for albums?

Reply:
Yes! I have no idea how I didn't catch that; fixed and updated the tests to assert this behavior. Thanks Kate